src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Circle extends Shape implements Printable {
```

Comment:
```suggestion
public class Circle extends Shape {
```
let's implement Printable in the Figure class